lib/SIL/LinearLifetimeChecker.cpp

```diff
@@ -205,8 +205,11 @@ void State::initializeConsumingUse(BranchPropagatedUser consumingUser,
   error.handleOverConsume([&] {
     llvm::errs() << "Function: '" << value->getFunction()->getName() << "'\n"
                  << "Found over consume?!\n"
-                 << "Value: " << *value << "User: " << *consumingUser
-                 << "Block: bb" << userBlock->getDebugID() << "\n\n";
+                 << "Value:\n";
+    value->printInContext(llvm::errs());
+    llvm::errs() << *value << "User:\n";
```

Comment:
This is fine to me. My only request is can you keep each logical print out on the same line. That is, can you put User on a new line with a new invocation of llvm::errs()?

Reply:
Oh yes, I've actually already printed `value` on the previous line, so this line should just be
```swift
llvm::errs() << "User:\n";
```

Reply:
Fixed!